identity/app/views/profile/privacyForm.scala.html

```diff
@@ -25,6 +27,12 @@
                 <p>@privacyForm.errors.map(formError => (formError.key, formError.message)).mkString(", ")</p>
             </div>
         }
+        @if(consentsUpdated) {
+            <div>
+                <h2 class="manage-account-header">Thank you</h2>
+                <p class="manage-account-headerNote">Your Email preferences have been updated.</p>
+            </div>
+        }
```

Comment:
not sure if there's a success version of the error box to show this instead, right now it looks a bit jammed in there  :(

Reply:
aaa we do!! can you use `.form_success`? and no classnames on `<h2>` or `<p>`, that should look a bit neater

Reply:
I think I'll need your help with this... I just tried it and it wasn't displayed at all when I changed the div class to `form__success`... I couldn't find `form_success`